docs/components/gantry/oneaxis.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Configure a One-Axis Gantry"
+linkTitle: "oneaxis"
+weight: 70
+type: "docs"
+description: "Configure a oneaxis gantry."
+tags: ["gantry", "components"]
+# SME: Rand
+---
+
+Configure a `oneaxis` gantry to integrate a one-axis gantry into your robot:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your sensor, select the type `gantry`, and select the `oneaxis` model.
+
+Click **Create component**.
+Paste into the **Attributes** box:
+
+``` json
+"motor": <"your-motor-name">,
+"length_mm": <int>,
+"axis": {
+    "X": <int>,
+    "Y": <int>,
+    "Z": <int>
+}
+```
+
+![Creation of a one-axis gantry component in the Viam app config builder.](../img/oneaxis-ui-config.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    // your motor config
+    {
+      "name": <"your-oneaxis-gantry-name">,
+      "type": "gantry",
+      "model": "oneaxis",
+      "attributes": {
+        "motor": <"your-motor-name">,
+        "length_mm": <int>,
+        "axis": {
+          "X": <int>,
+          "Y": <int>,
+          "Z": <int>
+        }
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+The following attributes are available for `oneaxis` gantries:
+
+| Attribute | Inclusion | Description |
+| ----------- | -------------- | --------------  |
+| `length_mm` | **Required** | Length of the axis of the gantry in millimeters. |
+| `motor` | **Required** | `name` of the [motor](/components/motor) that moves the gantry's actuator. |
```

Comment:
```suggestion
| `motor` | **Required** | Name of the [motor](/components/motor) that moves the gantry's actuator. |
```
[opt] For consistency. I haven't been stylizing these in other component docs. I guess we could change them all? Same applies below.

Reply:
Have been changing them, it helps as it should be same JSON string

Reply:
Makes sense! Cool!